napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -930,6 +943,12 @@ def _update_count_in_label(self):
             trans._("Available Plugins ({count})", count=count)
         )
 
+    def _end_refresh(self):
+        refresh_state = self.refresh_state
+        self.refresh_state = RefreshState.DONE
+        if refresh_state == RefreshState.Outdated:
```

Comment:
Tests are failing here @potating-potato!

https://github.com/napari/napari/runs/5595912701?check_suite_focus=true#step:9:78

Reply:
those typos! thanks for catching it!